### PR TITLE
Added links for becoming Approver and Maintainer

### PR DESCRIPTION
**Pull request by NitishKumar06**
Fix: #1112

### CODE REVIEW

File: content/en/contributor-ladder/_index.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Please refer to the [How to Contribute](/contribute/) and [Style Guide](/style-g
 
 ## 2. Approvers
 
-Approvers provide feedback on PRs and approve them. Any active contributor can become an approver (see "Becoming an approver"). 
+Approvers provide feedback on PRs and approve them. Any active contributor can become an approver (see [Becoming an approver](https://glossary.cncf.io/contributor-ladder/#becoming-an-approver)).
```
Comment (jihoon-seo) [inline content/en/contributor-ladder/_index.md:32]:
```suggestion
Approvers provide feedback on PRs and approve them. Any active contributor can become an approver (see [Becoming an approver](#becoming-an-approver)). 
```

File: content/en/contributor-ladder/_index.md [hunk 2]
```diff
@@ -73,7 +73,7 @@ This decision will be based on your demonstrated level of proficiency and respon
 
 ## 3. Maintainers
 
-Maintainers are approvers who can also merge PRs. Anyone can become a Glossary maintainer (see "Becoming a maintainer"). 
+Maintainers are approvers who can also merge PRs. Anyone can become a Glossary maintainer (see [Becoming a maintainer](https://glossary.cncf.io/contributor-ladder/#becoming-a-maintainer)).
```
Comment (jihoon-seo) [inline content/en/contributor-ladder/_index.md:76]:
```suggestion
Maintainers are approvers who can also merge PRs. Anyone can become a Glossary maintainer (see [Becoming a maintainer](#becoming-a-maintainer)). 
```

**Review by iamNoah1 — APPROVED** [review]
Thanks @NitishKumar06 
lgtm

**Review by CathPag — APPROVED** [review]
LGTM

**Pull request merged**